apps/vanilla/src/script/init.ts

```diff
@@ -0,0 +1,42 @@
+import { PageHandler } from '../namespaces/PageHandler';
+import { updateTable } from '../services/fetchAnime';
+
+import { ANIME_ROUTE, DEFAULT_QUERIES, NUMBER_OF_COLUMNS } from './constants';
+import { generateUrl } from './generateUrl';
+import { renderFilterOptions, renderOrderOptions, renderPagination, renderSortOptions } from './renderToDOM';
+
+/** Init event listener for pagination and render it to DOM. */
+export function initPagination(): void {
+  const firstPageButton = document.querySelector('.first-page-button');
+  firstPageButton?.addEventListener('click', PageHandler.goToFirstPage);
+  const lastPageButton = document.querySelector('.last-page-button');
+  lastPageButton?.addEventListener('click', PageHandler.goToLastPage);
+  renderPagination(0, 0);
+}
+
+/** Init query option. */
+export function initQuery(): void {
+  renderOrderOptions();
+  renderSortOptions();
+  renderFilterOptions();
+}
+
+/** Init anime table. */
+export function initAnimeTable(): void {
+  const tableRow = document.querySelector('.table__row');
+  if (tableRow !== null) {
+    for (let i = 0; i < NUMBER_OF_COLUMNS; i++) {
+      const loadingImage = document.createElement('img');
+      loadingImage.src = 'https://img.idesign.vn/2018/10/23/id-loading-1.gif';
+      loadingImage.alt = 'Loading...';
+      loadingImage.classList.add('table__row-image');
+      const cell = document.createElement('td');
+      cell.classList.add('table__row-content');
+      cell.append(loadingImage);
+      tableRow.append(cell);
```

Comment:
Please group your, even just one line helps to read and understand it.
In this example, we have 3 steps:
1. create image
2. create cell
3. use created cell

You will learn it with more experience, but even so try to put more thought into that.
```suggestion
      const loadingImage = document.createElement('img');
      loadingImage.src = 'https://img.idesign.vn/2018/10/23/id-loading-1.gif';
      loadingImage.alt = 'Loading...';
      loadingImage.classList.add('table__row-image');
     
      const cell = document.createElement('td');
      cell.classList.add('table__row-content');
      cell.append(loadingImage);
      
      tableRow.append(cell);
```